roles/cluster/tasks/main.yml

```diff
@@ -26,6 +26,21 @@
 - name: Run xapi-clusterd
   systemd: enabled=yes name=xapi-clusterd
 
+# Disabling the crashdump so that xensource.log entries in the physical host accurately reflect when nodes fenced
+#- name: Disable crashdump
+#  command: "sed -i 's/multiboot2\\(.*\\) crashkernel=.*,below=[0-9]\\+[A-Z]\\+\\(.*\\)/multiboot2\\1\\2/' /boot/grub/grub.cfg"
```

Comment:
I think this is not needed anymore because you are now parsing the timestamp of when the Xen watchdog fired based on the serial console, so we do not need the timestamps in the physical console log.

Still it is useful for documentation purposes to know how to disable the crashdumps should we need that in the future.